src/pages/ProfilePage.vue

```diff
@@ -0,0 +1,34 @@
+ 
+<template>
+  <div>
+    <Header />
+    <section class="hero is-small">
+      <div class="hero-body">
+        <div class="columns is-centered">
+          <div class="column is-5-tablet is-2-desktop">
+            <p class="subtitle">Welcome {{ stored.firstname }}</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+</template>
+<script>
+import Header from "../components/Header.vue";
+export default {
+  name: "ProfilePage",
+  data() {
+    return {
+      stored: [],
```

Comment:
default type array and on mount object is assign, please provide null if not sure of type and also can be read more for knowledge as still no error in console.